zombienet/0003-block-building-warp-sync/test-block-building-warp-sync.zndsl

```diff
@@ -0,0 +1,35 @@
+Description: Warp sync
+Network: ./test-block-building-warp-sync.toml
+Creds: config
+
+alice: is up
+bob: is up
+charlie: is up
+dave: is up
+
+alice: reports node_roles is 4
+bob: reports node_roles is 4
+charlie: reports node_roles is 1
+dave: reports node_roles is 1
+
+alice: reports peers count is at least 3 within 60 seconds
+bob: reports peers count is at least 3 within 60 seconds
+charlie: reports peers count is at least 3 within 60 seconds
+dave: reports peers count is at least 3 within 60 seconds
+
+
+# db snapshot has 12133 blocks
+dave: reports block height is at least 1 within 60 seconds
+dave: reports block height is at least 12132 within 60 seconds
```

Comment:
Is this the block we warp sync to? Or why checking for this particular block?

Reply:
db has 12133 blocks.